dp/cloud/python/magma/db_service/tests/unit/test_b0cad5321c88.py

```diff
@@ -0,0 +1,38 @@
+from magma.db_service.tests.alembic_testcase import AlembicTestCase
+
+
+class Testb0cad5321c88TestCase(AlembicTestCase):
+
+    def setUp(self) -> None:
+        super().setUp()
```

Comment:
It is possible to create different kind of test:
- specify some data before migration (that would be some records in the database)
- specify expected data after migration

For example suppose that we have the following table:
```
CREATE TABLE T (ID INT);
```
And the following records:
```
INSERT INTO T VALUES(1);
INSERT INTO T VALUES(2);
```
now when we execute the following migration:
```
ALTER TABLE T ADD COLUMN A INT;
UPDATE T SET A = ID;
```
then we should get the following result:
```
SELECT * FROM T
ID, A
1, 1
2, 2
```

So the question is can we do it with this version of a framework?